thorlcr/activities/diskread/thdiskreadslave.cpp

```diff
@@ -412,6 +412,7 @@ class CDiskReadSlaveActivity : public CDiskReadSlaveActivityRecord
         else
             limit = (rowcount_t)helper->getRowLimit();
```

Comment:
Not part of this pull request - but these can't be called in the constructor, they need to be in start()